site/content/api/Command-And-Control-API.md

```diff
@@ -72,4 +68,4 @@ The status property must contain a valid HTTP status code:
 
 For status codes indicating an error (codes in the `400 - 499` range) the message body MAY contain a detailed description of the error that occurred.
 
-If a control message response contains some additional diagnostic information, the body of the message MUST consist of a single AMQP *Data* section containing a control message data. 
+If a command message response contains some additional diagnostic information, the body of the message MUST consist of a single AMQP *Data* section containing the response message data. 
```

Comment:
I wouldn't call it *diagnostic information* but simply the response payload, right?